math/complex_numbers.cpp

```diff
@@ -0,0 +1,238 @@
+/**
+ * Copyright 2020 @author tjgurwara99
```

Comment:
```suggestion
 * @author tjgurwara99
```
No need to add "copyright message". The repo already has an associated license.